test_helpers/utils.go

```diff
@@ -24,6 +24,19 @@ func ConnectWithValidation(t testing.TB,
 	return conn
 }
 
+func DeleteRecordByKey(conn tarantool.Connector, space interface{}, index interface{}, key []interface{}) {
+	req := tarantool.NewDeleteRequest(space).
+		Index(index).
+		Key(key)
+	resp, err := conn.Do(req).Get()
+	if err != nil {
+		panic("Failed to Delete: " + err.Error())
+	}
+	if resp == nil {
+		panic("Response is nil after Select")
```

Comment:
Please, try to avoid `panic` here, reason:
https://github.com/tarantool/go-tarantool/issues/147

Reply:
I guess it is best I can do:

https://github.com/tarantool/go-tarantool/blob/6b08e7bac3df8d13fb0b6ec36310fcfc08d61f65/test_helpers/utils.go#L28-L39

Reply:
It may be better to add a `t testing.TB` as a first arg and to use t.Helper() + t.Fatalf(). See other functions in utils.go.

Reply:
Thank you! Fixed:

https://github.com/tarantool/go-tarantool/blob/3a86a3f9c38290154195b716d207ae9493335797/test_helpers/utils.go#L27-L41